src/commands/dev.ts

```diff
@@ -0,0 +1,297 @@
+import { flags } from '@oclif/command';
+import axios, { AxiosResponse } from 'axios';
+import chalk from 'chalk';
+import execa from 'execa';
+import fs from 'fs-extra';
+import isCi from 'is-ci';
+import yaml from 'js-yaml';
+import opener from 'opener';
+import { EnvironmentUtils } from '../architect/environment/environment.utils';
+import Command from '../base-command';
+import LocalDependencyManager from '../common/dependency-manager/local-manager';
+import { DockerComposeUtils } from '../common/docker-compose';
+import DockerComposeTemplate from '../common/docker-compose/template';
+import DeployUtils from '../common/utils/deploy.utils';
+import * as Docker from '../common/utils/docker';
+import { ComponentSlugUtils, ComponentSpec, ComponentVersionSlugUtils } from '../dependency-manager/src';
+import { buildSpecFromPath } from '../dependency-manager/src/spec/utils/component-builder';
+
+export abstract class DevCommand extends Command {
+
+  static flags = {
+    ...Command.flags,
+  };
+
+  parse(options: any, argv = this.argv): any {
+    const parsed = super.parse(options, argv);
+    const flags: any = parsed.flags;
+
+    // Merge any values set via deprecated flags into their supported counterparts
+    parsed.flags = flags;
+
+    return parsed;
+  }
+}
+
+export default class Dev extends DevCommand {
+  auth_required(): boolean {
+    return false;
+  }
+
+  static description = 'Run your stack locally';
+
+  static flags = {
+    ...DevCommand.flags,
+    ...EnvironmentUtils.flags,
+
+    compose_file: flags.string({
```

Comment:
Since its a new command can we remove the deprecated flags?

Reply:
Yeah we can deprecate on this command, and for the proxy on the other I can just translate to the new one.

Reply:
So turns out proxying the commands is not as simple as I thought. Instead I added to warning all deprecated flags will be removed and have them hidden.